.github/workflows/link-existing-jira-issue.yml

```
@@ -0,0 +1,97 @@
+
+jobs:
+    link-jira:
+        # qa 라벨이 있을 때만 Jira 연동 로직 실행
+        if: contains(github.event.issue.labels.*.name, 'qa')
+        name: Link Existing Jira Issue
+        runs-on: ubuntu-latest
+
+        steps:
+            -   name: Extract Jira Key
+                id: extract
+                run: |
```

actionlint (1.7.8)

"github.event.issue.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)